### DISCUSSION TITLE
Inheriting fields from class are not visible in the generated schema

### DISCUSSION THREAD

**Discussion by labamba63** [graphql-java/graphql-java#2389] [Q&A]
Hi!

I changed to the new version and now when generating the schema, inherited fields in the schema do not appear to me. Does anyone know how it's done now?

Regards.

**Comment by labamba63** [comment]
Adding a package is the solution.
`GraphQLSchemaGenerator()
                .withBasePackages(...)`

**Comment by bbakerman** [comment]
I am pretty sure this is a problem in a library built on top of graphql java
